sdk/storage/CMakeLists.txt

```diff
@@ -41,17 +41,19 @@ set (AZURE_STORAGE_DATALAKE_HEADER
     inc/common/storage_uri_builder.hpp
     inc/common/common_headers_request_policy.hpp
     inc/common/shared_key_policy.hpp
-    inc/common/token_credential_policy.hpp
     inc/common/crypt.hpp
     inc/common/constants.hpp
     inc/datalake/datalake.hpp
     inc/blobs/internal/protocol/blob_rest_client.hpp
     inc/datalake/protocol/datalake_rest_client.hpp
+    inc/datalake/response_models.hpp
```

Comment:
This doesn't align with `datalake_options.hpp`. Either `datalake_response_models.hpp` or `options.hpp`

Reply:
But I feel `datalake_response_models.hpp` is kind of long. How about `datalake_responses.hpp`?

Reply:
Well, we talked offline before I pinned down the name `response_models`, but I don't really think it matters too much so I'll change it in next iteration.